### ISSUE TITLE
Barba.js with MVC 5 Form Validation

### ISSUE THREAD

**Issue by TrevorFourie** [luruke/barba.js#330]
Hi,

Thank you for a great plugin! Really appreciate the time and effort with this and sharing.

I was wondering if anyone could assist with an issue I'm having. 

I have a MVC 5 web site. I'm using Barba.js as smooth transition.

When I go to the contact form the validation does not work. When i refresh the contact form page, it works. 

The validation requirements are in the Modal and I'm using unobtrusive ajax validation.

I've tried to reload the scripts as well

Barba.Dispatcher.on('newPageReady', function (currentStatus, oldStatus, container) {
            $.getScript("/content/scripts/custom/jquery.unobtrusive-ajax.js");
            $.getScript("/content/scripts/custom/jquery.validate.js");
            $.getScript("/content/scripts/custom/jquery.validate.unobtrusive.js");
            $.validator.unobtrusive.parse(document);
        });

As well as $("#formcontact").validate()

As you can see the validation parameters are there but it's just not validating?
 

> <div class="form-group">
    <div class="col-12">Contact Number</div>
    <div class="col-12"><input autocomplete="off" class="form-control ddv-form" data-val="true" data-val-length="The Contact Number seems to be missing some numbers." data-val-length-max="15" data-val-length-min="10" data-val-maxlength="The Contact Number seems to be invalid" data-val-maxlength-max="15" data-val-phone="The Contact Number field is not a valid phone number." data-val-regex="Entered phone format is not valid." data-val-regex-pattern="^[+ ]?\(?([0-9]{2,5})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$" data-val-required="Please provide a contact number." id="Telephone" label="Contact Number" name="Telephone" placeholder="Contact Number" tabindex="2" title="Please provide a contact number" type="text" value="" /></div>
    <div class="col-12"><span class="field-validation-valid text-danger" data-valmsg-for="Telephone" data-valmsg-replace="true"></span></div>
</div>

Can anyone maybe assist me in telling me what I'm missing?

Thank you for your time.

**Comment by luruke** [comment]
Hello @TrevorFourie , i'm sorry but I can't help you on this.
In any case it's not a specific issue of barba.
